### PR TITLE
Fix broken formatting in Installing guide

### LINKED ISSUE #4477: Fix broken formatting in Installing guide
Review the webpages in the Installing guide and fix formatting issues such as:
- incorrect bullet points or numbering
- broken tab formatting

The following issues need to be fixed:
- [x] https://knative.dev/docs/install/installing-istio/#choosing-an-istio-installation “Installing Istio without sidecar injection(Recommended default installation)” missing a space before the opening parenthesis.
- [x] https://knative.dev/docs/install/installing-istio/#verifying-your-istio-install incorrect formatting for the tip. Correct formatting uses the opening tip tag (`!!! tip`), followed by a new line, and then 4 spaces before the tip text. 
For example:
    ```
    !!! tip
        Tip text …
    ```
- [x]  https://knative.dev/docs/install/upgrade/upgrade-installation-with-operator/#rollback-to-an-earlier-version tab formatting is broken. To fix, indent the lines 126-146 and 150-170 by an extra 4 spaces.
- [x] https://knative.dev/docs/install/uninstall/#uninstalling-optional-serving-extensions Broken numbering in the "TLS with cert-manager" tab. To fix, check for correct indentation of the code snippet (8 spaces).
- [x] https://knative.dev/docs/install/uninstall/#uninstalling-optional-eventing-extensions broken numbering in Apace Kafka sink tab. To fix, check for correct indentation of the code snippet (8 spaces).
- [ ] **Extra Credit:** Add code tags around the names of the YAML files in the tables on the following pages:
    - https://knative.dev/docs/install/yaml-install/serving/serving-installation-files/
    - https://knative.dev/docs/install/yaml-install/eventing/eventing-installation-files/

### PR DESCRIPTION
**Pull request by Subhasish-Behera**
fixes #4477

### CODE REVIEW

File: docs/install/installing-istio.md [hunk 1]
```diff
@@ -159,9 +159,9 @@ all of the pods show a `STATUS` of `Running` or `Completed`:
 ```bash
 kubectl get pods --namespace istio-system
 ```
-
-> Tip: You can append the `--watch` flag to the `kubectl get` commands to view
-> the pod status in realtime. You use `CTRL + C` to exit watch mode.
+!!! tip
+>   Tip: You can append the `--watch` flag to the `kubectl get` commands to view
+>   the pod status in realtime. You use `CTRL + C` to exit watch mode.
```
Comment (snneji) [inline docs/install/installing-istio.md:164]:
```suggestion
    You can append the `--watch` flag to the `kubectl get` commands to view
    the pod status in realtime. You use `CTRL + C` to exit watch mode.
```

**Review by snneji — CHANGES_REQUESTED** [review]
Only one small tweak. Other than that it looks great. Thanks!

**Comment by Subhasish-Behera** [comment]
done

**Comment by snneji** [comment]
Thanks for doing this one @Subhasish-Behera. 🙂

/lgtm
/approve

**Pull request merged**